### PR TITLE
repo: Prepare release (`1.19.4`)

### PR DESCRIPTION
**Pull request by phlax**
Signed-off-by: Ryan Northey <ryan@synca.io>

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Review by zuercher — APPROVED** [review]

**Comment by phlax** [comment]
im gonna keep this wip until any other changes - like CI fixes have landed

File: docs/root/version_history/v1.19.4.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
+1.19.4 (April 12, 2022)
```
Comment (phlax) [inline docs/root/version_history/v1.19.4.rst:1]: adjust date before landing

File: docs/root/version_history/current.rst [hunk 2]
```diff
@@ -9,8 +9,6 @@ Minor Behavior Changes
 ----------------------
```
Comment (phlax) [inline docs/root/version_history/current.rst:9]: im not sure what current version in docs should be - ill check historically

**Comment by phlax** [comment]
docs are rendered here https://storage.googleapis.com/envoy-pr/20795/docs/index.html

**Comment by phlax** [comment]
@pradeepcrao i dont see any other backport requests or issues for v1.19, and this is the only ticket in the milestone, so im thinking it should be good to land and tag the release

cc @mattklein123

**Pull request merged**